rust/benches/file_backend.rs

```diff
@@ -267,38 +267,41 @@ fn file_backend_benchmark_matrix(c: &mut Criterion) {
     let target_file_size = if cfg!(debug_assertions) {
         100_000
     } else {
-        let size = memory_kb * 1024 * FILE_SIZE_MEMORY_MULTIPLIER;
-        eprintln!(
-            "Using benchmark file of size {} * main memory size ({} GiB) = {} GiB to limit effects of OS page cache",
-            FILE_SIZE_MEMORY_MULTIPLIER,
-            memory_kb / 1024 / 1024,
-            size / ONE_GB as u64
-        );
-        size
+        memory_kb * 1024 * FILE_SIZE_MEMORY_MULTIPLIER
     };
 
     let path = Path::new(FILE);
+    let init_file_fn = || {
+        static INIT: Once = Once::new();
+        INIT.call_once(|| {
```

Comment:
We would also use a global static `LazyLock` which evaluates to `()`. This way we don't need to pass the closure around.